### ISSUE TITLE
Inconsistent values for basisOfRecord, like MaterialSample/Sample

### ISSUE THREAD

**Issue by jeroencreuwels** [gbif/portal-feedback#1396]
In addition to various other issues in Github (see list below) on the issue of basisOfRecord values, I would like to add the confusion on  MaterialSample as a choice. For biodiversity data that describes the outcomes of material sampling, without reference to physically preserved material, the option MaterialSample  seems a good choice.

If a user of the IPT unmaps the field basisOfRecord and chooses from the dropdown list, the option MaterialSample is not visible. Here, only “Sample” is available. When clicking on the “i” left of “basisOfRecord” it is even not mentioned that Sample/MaterialSample is a possibility. 

I noticed that in the IPT there at least there are place where information is given on the Type Vocabulary:   
a)http://rs.gbif.org/vocabulary/dwc/basis_of_record.xml
b)http://rs.tdwg.org/dwc/terms/index.htm#basisOfRecord 
c)…../ipt/manage/http://rs.tdwg.org/dwc/dwctype/  
with different content.

Maybe it is advisable to achieve more consistency in basisOfRecord values, and definitively for 
MaterialSample. 

Other issues about this topic:
Improve basis of record facets #284
Consistency of basisOfRecord everywhere #1311
Revise accepted values for basisOfRecord #14

**Comment by MortenHofft** [comment]
I think `Revise accepted values for basisOfRecord #14` is referencing a wrong issue? I don't see it has anything to do with Basis Of Record?

`Improve basis of record facets` should go here https://github.com/gbif/portal16/issues/284

I'm not sure how to label this as it is a cross cutting concern, but I guess `API` is the best match

**Comment by tucotuco** [comment]
I agree that consistency is paramount, and that the recommendation should
be exactly the following, with no variations on spelling or spacing or
capitalization:

PreservedSpecimen
LivingSpecimen
FossilSpecimen
Occurrence (but avoid in favor of the above if possible)

MaterialSample

HumanObservation
MachineObservation
Event (but avoid in favor of the above if possible)

And as @mdoering mentioned in
https://github.com/gbif/portal-feedback/issues/87, officially work out the
ontology of terms.
